gradle/suppress.gradle

```diff
@@ -0,0 +1,29 @@
+import groovy.xml.XmlUtil
+
+def suppressCves(reportPath, suppressionsPath) {
+    def report = new XmlSlurper().parse(new FileReader(reportPath))
+    def vulns = report.'**'.findAll { node -> node.name() == 'vulnerability' }.collect { it.name.text() } as Set<String>
```

Comment:
This grabs the unique set of non-suppressed vulnerabilities from the output of a dependency check report - there can be duplicates in the report.